src/spatialdata/models/_utils.py

```diff
@@ -167,9 +167,9 @@ def _(e: MultiscaleSpatialImage) -> tuple[str, ...]:
 
 @get_axes_names.register(GeoDataFrame)
 def _(e: GeoDataFrame) -> tuple[str, ...]:
-    all_dims = (X, Y, Z)
+    all_dims = (Z, Y, X)
```

Comment:
@timtreis @Sonja-Stockhaus  I restored x, y, z since this is what also we do for points and since other parts of the framework (like napari) expect this. But we can consider changing it in another PR, what motivated the change here?

Reply:
Honestly don't remember, has been weeks. Either consistency with sth else or a bug